srcpkgs/libopenmpt/template

```diff
@@ -1,33 +1,32 @@
 # Template file for 'libopenmpt'
 pkgname=libopenmpt
-version=0.5.4
+version=0.5.5
 revision=1
 wrksrc="libopenmpt-${version}+release.autotools"
 build_style=gnu-configure
-configure_args="$(vopt_with pulseaudio)
- $(vopt_with sdl) $(vopt_with sdl2)
- $(vopt_enable libopenmpt_modplug) $(vopt_enable libmodplug)"
-hostmakedepends="pkg-config"
-makedepends="zlib-devel mpg123-devel libogg-devel libvorbis-devel
- portaudio-devel portaudio-cpp-devel libsndfile-devel libflac-devel
+configure_args="$(vopt_with pulseaudio) $(vopt_with sdl2) $(vopt_with portaudio)"
+hostmakedepends="pkgconf"
```

Comment:
Use `pkg-config` as other templates already do.

Reply:
This is convention now? Hm, whatever.